OpenSim/Simulation/Model/ContactHalfSpace.cpp

```diff
@@ -59,4 +60,31 @@ SimTK::ContactGeometry ContactHalfSpace::createSimTKContactGeometry() const
     return SimTK::ContactGeometry::HalfSpace();
 }
 
+void ContactHalfSpace::generateDecorations(bool fixed, const ModelDisplayHints& hints,
+    const SimTK::State& s,
+    SimTK::Array_<SimTK::DecorativeGeometry>& geometry) const
+{
+    Super::generateDecorations(fixed, hints, s, geometry);
+
+    // There is no fixed geometry to generate here.
+    if (fixed) { return; }
+
+    // B: base Frame (Body or Ground)
+    // F: PhysicalFrame that this ContactGeometry is connected to
+    // P: the frame defined (relative to F) by the location and orientation
+    //    properties.
+    if (hints.get_show_contact_geometry()) {
+        const auto& X_BF = getFrame().findTransformInBaseFrame();
+        const auto& X_FP = getTransform();
+        const auto X_BP = X_BF * X_FP;
+        geometry.push_back(SimTK::DecorativeBrick(Vec3{.005, 0.5, 0.5})
```

Comment:
Old visualizer was using a mesh file that has arrows, We can use that here instead of the Brick but stretching could be undesirable. Other possible options are: Make the material one-sided so the top shows but the bottom doesn't, or use different materials/textures/colors for top/bottom.